src/metatensor/models/cli/export_model.py

```diff
@@ -37,20 +34,12 @@ def _add_export_model_parser(subparser: argparse._SubParsersAction) -> None:
     )
 
 
-def export_model(model: torch.nn.Module, output: str) -> None:
-    """Export a pre-trained model to run MD simulations
+def export_model(model, output):
+    """Exports a trained model to allow it to make predictions,
+    including within molecular simulation engines.
 
-    :param model: Path to a saved model
+    :param model: Path to a saved model checkpoint
     :param output: Path to save the exported model
     """
 
-    for model_output_name, model_output in model.capabilities.outputs.items():
-        if model_output.unit == "":
-            warnings.warn(
-                f"No units were provided for the `{model_output_name}` output. "
-                "As a result, this model output will be passed to MD engines as is.",
-                stacklevel=1,
-            )
-
-    wrapper = MetatensorAtomisticModel(model.eval(), model.capabilities)
-    wrapper.export(output)
+    export(model, output)
```

Comment:
Does this also work if the user provides an already exported model or will he get a cryptic error message?